go/logictest/resultparser.go

```diff
@@ -67,7 +68,28 @@ func ParseResultFile(f string) ([]*ResultLogEntry, error) {
 	}
 }
 
-func parseLogEntry(scanner *parser.LineScanner) (*ResultLogEntry, error) {
+func ParseResultFileWithDuration(f string) ([]*ResultLogEntry, error) {
+	file, err := os.Open(f)
+	if err != nil {
+		panic(err)
+	}
+
+	var entries []*ResultLogEntry
+
+	scanner := parser.LineScanner{Scanner: bufio.NewScanner(file)}
+
+	for {
+		entry, err := parseLogEntry(&scanner, true)
+		if err == io.EOF {
+			return entries, nil
+		} else if err != nil {
+			return nil, err
+		}
+		entries = append(entries, entry)
+	}
+}
+
+func parseLogEntry(scanner *parser.LineScanner, lineHasDurations bool) (*ResultLogEntry, error) {
```

Comment:
No need to toggle this behavior, just always output the duration. We don't need to be able to parse old files.